src/main/java/com/netgrif/application/engine/elastic/service/ElasticCaseService.java

```diff
@@ -41,25 +42,28 @@
 @Service
 public class ElasticCaseService extends ElasticViewPermissionService implements IElasticCaseService {
 
-    private static final Logger log = LoggerFactory.getLogger(ElasticCaseService.class);
+    protected static final Logger log = LoggerFactory.getLogger(ElasticCaseService.class);
```

Comment:
Why protected? Maybe getters for them would be considerable.

Reply:
Done